cmd/server/server.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"log"
+	"log/slog"
+	"net/http"
+
+	"github.com/ryohei1216/go-mvc/cmd/router"
+	"github.com/ryohei1216/go-mvc/db"
+)
+
+func main() {
+	db, err := db.New()
+	if err != nil {
+		slog.Warn(err.Error())
+		panic("failed to connect database")
+	}
+
+	r := router.New(db)
+
+	log.Fatal(http.ListenAndServe(":8080", r))
```

Comment:
こいつのエラーをslogに流してないのって何か意図あったりします？(せっかくslogというナウいライブラリを使ってるので、こういうところも全部構造化ログに流してしまって良いかなと思いました！)

あとechoサーバーの時にコメントするか迷ってしなかったのですが、log.Fatalはlogのパッケージのくせにpanicを内部で呼び出してプログラムが強制終了するという命名からは想像がつかない副作用があるので、他社のチームによってはそもそも使うこと自体を禁止しているところもあるそうです。今回の場合はシンプルにerrが帰ってきたらpanicを呼び出すくらいでも良い気はしました！

Reply:
ListenAndServe()が失敗した時点でアプリを落としたいと思いlog.Fatal()を使用しています！（自分もmain関数など最上位レイヤーでしか使用しないように気をつけています 🙏 ）
slogを使わなかったのはos.Exit(1)でログと同時にアプリを落としたかったんですが、slogにはデフォルトでこれに相当するものがないのでlog.Fatal()を使用していました！

Reply:
なるほど！「標準出力をした上でpanicをする」という仕様を満たしたいのであれば14行目と同じ書き方で、
```suggestion
	if err := log.Fatal(http.ListenAndServe(":8080", r)); err != nil {
	  slog.Error(err.Error())
	  panic(err)
	}
```
で書けそうですかね？
同じ仕様を満たすコードの書き方はなるべく統一した方が認知負荷が下がるのでよいかな？と思いました！

Reply:
(サンプルのコメント)
[9ed8f22](https://github.com/ryohei1216/go-mvc/pull/1/commits/9ed8f22883d364bfee021c8a6012242c7fdd0cdd)で修正